docs/tokenomics/token-holders/nns-app-quickstart.md

```diff
@@ -264,6 +264,54 @@ To disburse a neuron and return its ICP utility tokens:
 
 - #### Step 4:  Verify the transaction information, then click **Confirm and Send**. 
 
+## Neuron following
+
+To maximize neuron rewards and vote on every proposal submitted to the NNS, neuron holders can make use of the NNS's liquid democracy. It may be hard for neuron owners to manually vote on every single proposal submitted or neuron owners may not have the necessary level of technical understanding to evaluate proposals themselves. To aid in this, the NNS supports **neuron following**.
+
+A neuron can be configured to vote automatically by following the voting decision made by a group of other neurons. Following is done on the basis of topics, such that a neuron can choose to follow the voting decision for different proposal topics, such as governance or subnet management. For example, if the neuron has been configured to follow a group's decision for governance proposals, if the group votes to adopt a governance proposal with more than half of the followees voting in favor of adoption, the neuron following that group will vote to approve the proposal as well. If the neuron has been configured to follow a group's decision for subnet management proposals, if the group votes to reject a subnet management proposal with more than half of the followees voting to reject, the neuron following the group will vote to reject the proposal. 
+
+:::caution
+It is important to note that when following a group of neurons, your neuron will only cast a vote if at least 6 out of 10 of the neurons within the followed group vote the same way. If only 5 neurons vote, your neuron will not cast a vote. Therefore, it is recommended to follow groups with at least 10 followees.
+:::
+
+Additionally, there is a catch-all follow rule that may be defined for a neuron to automatically vote on proposals for which no follow rule has been defined; however, this catch-all follow rule does not include SNS & neuron's fund or governance proposals. It is in each neuron's interest to follow other neurons that support the best interests of the network and their own economic interests.
+
+To configure your neuron to follow the group of another neuron:
+
+- #### Step 1: From the NNS dapp, select 'My Neuron Staking', then select the neuron that you'd like to set a following rule for.
+
+![NNS Neuron following 1](../_attachments/following1.png)
+
+- #### Step 2: In the neuron's details, scroll down to the 'Following' section and select 'Follow Neurons'. 
+
+![NNS Neuron following 2](../_attachments/following2.png)
+
+- #### Step 3: Next, select a topic to follow. 
+
+The topic refers to the different types of proposals; for example, the 'Governance' topic defines a following role for governance type proposals, while the 'Subnet Management' topic defines a following role for subnet management proposals. There is also a 'catch-all' rule that can be followed for all proposal types except for governance, SNS, and neurons' fund. 
```

Comment:
```suggestion
The topic refers to the different types of proposals; for example, the 'Governance' topic defines a following role for governanc-related proposals, while the 'Subnet Management' topic defines a following role for subnet management proposals. There is also a 'catch-all' rule that can be followed for all proposal types except for governance, SNS, and neurons' fund. 
```

(types is a fixed term we use for something else, therefore I would not use it here)